### DISCUSSION TITLE
思考：开源运营中的 over marketing 行为

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/open-source-best-practice#127] [Open Source Cases]
开源与传统的商业行为不同，并不是可以通过造势、华丽的外表、大量资本注入就可以做好的。开源的大部分行为都会被永久留存，那么，在开源运营过程中有哪些可能会是：过度的市场行为（over marketing）、过度的修饰，而这些行为又会给开源社区（项目）本身带来什么影响呢？

## 案例1
项目 A，在正式版本 v4.0.0 发布前，运营人员希望在项目的 alpha 阶段，通过发布一个功能新特性的预告，让社区成员得以提前体验并反馈。在博客的文章中，有类似如下的一段表述：

`除上述功能外，有更多重量级功能会在正式版本发布后在 Release Notes 中详述`

首先，我想补充一个软件迭代以及版本的背景知识。一个软件项目，大致的迭代过程可以是：alpha、beta、rc，等功能稳定后发布 GA 版本。但是，通常会在 feature 开发完成后，才会进入到 alpha；此时，表明了功能基本完成，主要流程可以走通，但不够稳定、可能会有 bug 出现。因此，大概率不太会出现 alpha 版本之后还会有**更多重量级功能**进入到正式版本中。

其次，即使是想要给读者留下悬念，以吸引大家的注意力，但也应该能够自圆其说、确保语言表述不是错误的。

细心的开源参与者，是希望参与到严谨的开源社区（项目）中来。而对于开源项目的用户，也不要“轻易”相信“鼓吹”式宣传文案。

在这个案例中，我呼吁开源运营人员尽量在了解软件发布基本常识，并尊重事实的前提下，再发挥文字的“修饰”效果。宣传文案虽然不需要像源代码那么严谨，但至少要确保实事求是，不打插边球。

**Comment by LinuxSuRen** [comment]
案例2

![image](https://user-images.githubusercontent.com/1450685/165927239-1a65bc78-bcb6-4537-897b-b7e9090a8e93.png)

**Comment by LinuxSuRen** [comment]
案例3

[看看什么叫惊喜！| “Alluxio开源社区贡献积分奖励计划”正式上线](https://mp.weixin.qq.com/s?__biz=MzkxMzMyODAwMQ==&mid=2247483961&idx=1&sn=63e54598339d9ea54199c00897522f7f&scene=21#wechat_redirect)

**Comment by LinuxSuRen** [comment]
CSDN 旗下的 GitCode，全局广告，任何一个仓库都有“置顶”消息。
![image](https://user-images.githubusercontent.com/1450685/222999623-7c13ccbd-5bc9-4a1a-a91e-a4cd52859a6f.png)
![image](https://user-images.githubusercontent.com/1450685/222999661-4a3bed49-662a-4393-ade9-5028b0d76212.png)

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

没错, 其实就象: [utterances](https://utteranc.es/)
如果将 GH-Issue 看作开放的文档数据库, 那真心有很多可玩的了...

**Comment by LinuxSuRen** [comment]
[春松客服](https://github.com/cskefu/cskefu) 的负责人[hailiang-wang](https://github.com/hailiang-wang)以为 GitHub 公开邮件就默认接受推广信息事件。以下是相关链接：

* [各位在 GitHub 公开邮箱就代表你们愿意接受各种无关项目的推广信息么?](https://v2ex.com/t/969939)
* https://github.com/cskefu/cskefu/issues/931
* [直播讨论，来自老范讲故事 YouTube](https://www.youtube.com/watch?app=desktop&v=Khto6J7inCU)

**Comment by LinuxSuRen** [comment]
点过 Star 后才开放文档阅读

![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/b1be076b-0622-481b-8d20-b17ab81e7d0b)

https://sa-token.cc/doc.html#/
https://www.v2ex.com/t/1024935
https://twitter.com/LinuxSuRen/status/1769906852496879709

**Reply by wjsvec** [reply]
这个也太离谱了

**Comment by LinuxSuRen** [comment]
北京开物数智科技有限公司（以下称：开物数智）在其官网宣传中碰瓷三个开源项目：
* [浪潮 KaiwuDB 严正声明！](https://mp.weixin.qq.com/s/jHAUCh9obZTEp4KniUV4mg) 发布于 2025.5.20
* [映云科技严正声明](https://mp.weixin.qq.com/s/MEtv3Mm-KRa6CKb54KvkrA) 发布于 2025.5.21
* [零容忍！涛思数据发严正声明](https://mp.weixin.qq.com/s/ui3AZxQjsR1yYkH_Zi3bHw) 发布于 2025.5.20

其他相关媒体报道：
* [涛思数据与浪潮KaiwuDB商标被侵权引发开源商业化合规思考](https://segmentfault.com/a/1190000046558296)

开物数智成立于 2020-03-30，法人胡森标。
